catalogue.json

```diff
@@ -118,7 +118,7 @@
             {
                 "name": "RJSDMX",
                 "description": "rsdmx is a package to parse/read SDMX data and metadata in R. It provides a) a set of classes and methods to read data and metadata documents exchanged through the Statistical Data and Metadata Exchange (SDMX) framework. The package currently focuses on the SDMX XML standard format (SDMX-ML); and b) an interface to SDMX web-services for a list of well-known data providers, such as EUROSTAT, OECD, and others.",
-                "supported_features" : "This is an extended text to add more information about the tool, it could contain information about platforms etc.",
+                "supported_features" : "See link to the tool",
                 "owner": "Emanuel Blondel",
```

Comment:
A small typo on my name :-) maybe my fault when filling the form: Emmanuel Blondel